.coveragerc

```
@@ -0,0 +1,3 @@
+[run]
+omit = *_test.py,tests/*,generator/common/gx_schema.py,*__init__*,generator/vendor/*
```

```suggestion
omit = 
  tests/*
  generator/common/gx_schema.py
  generator/vendor/*
```

`coverage` supports this multi line format, which is easier to read. Also, some of the entries are unnecessary:

* there are no `*_test.py` files in the repo and
* there is only a single `__init__.py` file in the already omitted `generator/vendor/`. (If you really intend to exclude all future `__init__.py` files, I suggest to use the more specific `*/__init__.py` instead.)